src/services/Sns.js

```diff
@@ -0,0 +1,37 @@
+const fetch = require('node-fetch');
+const cheerio = require('cheerio');
+
+const url = 'https://www.sns24.gov.pt/alertas/';
+
+/**
+ * Get alerts data form SNS24 website
+ *
+ * @returns {Array} data
+ */
+const getAlerts = async () => {
+  const data = await fetch(url)
+    .then(res => res.text())
+    .then((body) => {
+      const html = body;
+      const $ = cheerio.load(html);
+      const alertsList = [];
+      const $alerts = $('article.alerta');
+
+      $alerts.each((i, el) => {
```

Comment:
Could this be a `.map()` instead of `.each()`? If so then we dismiss the intermediate variable `alertsList`;

```js
return $alerts.map((i, el) => (
  {
    title: $(el).find('h2.pag-alertas').text(),
    description: $(el).find('p.pag-alertas').text().trim(),
    link: $(el).find('a.pag-alertar').attr('href'),
    dateTime: $(el).find('time.updated').attr('datetime'),
  }
);
```

Reply:
here im not sure i need to test.

Reply:
Yes its possible , i looked into the docs and map its there :) , but in the end the get() method needs to be called.
`      return $alerts.map((i, el) => (
        {
          titlex: $(el).find('h2.pag-alertas').text(),
          description: $(el).find('p.pag-alertas').text().trim(),
          link: $(el).find('a.pag-alertar').attr('href'),
          dateTime: $(el).find('time.updated').attr('datetime'),
        }
      )).get();`

I will update the PR.